hack/build/cleanup-images-and-pipelineruns.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# PipelineRun builds images for application.
+# This script removes images from Openshift internal image registry
+# to have ability re-triger PipelineRuns without skip build image tasks.  
```

Comment:
Pipelines are having `rebuild` parameter when it's set to 'true' then the build is run always -> https://github.com/redhat-appstudio/build-definitions/blob/main/pipelines/base/template-build/template-build.yaml#L30